docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+
+![Manage gear displaying "DE' to indicate that the user's 'Demo' profile is active](images/profiles/profile-gear-two-letters.png)
+
+### Edit a profile
+
+You can edit a profile just as you would normally change any VS Code configuration. You can install/uninstall/disable extensions, change settings, and adjust snippets like normal. These changes are stored in your currently active profile.
+
+### Workspace associations
+
+When you select create or select an existing profile, it is associated with the current workspace and whenever you open that folder, the workspace's profile is active. If you open another folder, the profile either goes back to the Default Profile if no profile is set, or switches to that folder's associated profile.
+
+Setting a workspace's profile back to the Default Profile clears the workspace's profile association.
+
+## Managing profiles
+
+### Switch profiles
+
+You can quickly switch between profiles with the **Profiles: Switch Profile** command in the Command Palette, which presents a dropdown listing your available profiles.
+
+### Rename a profile
+
+You can rename an existing profile via the **Rename** command in the Profiles menu.
+
+### Delete a profile
+
+You can delete a profile via the **Delete Profile** command. The **Delete Profile** drop down lets you select which profile(s) to delete.
+
+## Profile contents
+
+The **Profiles: Show Contents** command (available in the Command Palette or Profiles menus) brings up the Profiles view, where you can review the customizations for the profile.
+
+![Profile view displaying contents of a Doc Work profile](images/profiles/profile-show-contents.png)
+
+A profile can include:
+
+* Settings - In a profile-specific `settings.json` file.
+* Extensions - You can remove extensions from the profile by unchecking their entry in the Profiles view.
+* UI state - Currently open views and their positioning.
+* Keybindings - In a profile-specific `keybindings.json` file.
+* Snippets - In a profile-specific `{language}.json` files.
+* User Tasks - In a profile-specific `tasks.json` file.
+
+When you create a new profile based on the Default Profile, the profile-specific configuration files are populated from your user configuration files. Workspace-specific settings are not automatically included in a new profile.
+
+## Share Profiles
+
+### Export
+
+You can export a profile in order to save it or share it with others. The **Export Profile** command displays the Profiles view with the contents of the active profile and an **Export** button. You can unselect various elements of the profile such as extensions or configuration files before you export the profile.
+
+When you select **Export**, you are prompted for the profile name and whether you want to export to a [GitHub gist](https://docs.github.com/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) or your local file system.
+
+#### Save as a GitHub gist
```

Comment:
Do we use #### in our docs? This does not get rendered nicely in vscode. Should we at least bold this small title for clarity?

Reply:
We recently tuned the H4 styling on the website to be a reasonable font size and bold. Here's what it will look like live:

![image](https://user-images.githubusercontent.com/12818376/221628376-15b30048-84c2-4a96-ac8d-bb458d44b777.png)